x-pack/plugins/cases/server/saved_object_types/migrations/configuration.ts

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Moved the configuration migrations into their own file.

Reply:
Thanks!